synthesis/codon/codon.go

```diff
@@ -71,6 +73,18 @@ func (e invalidAminoAcidError) Error() string {
 	return fmt.Sprintf("amino acid %q is missing from codon table", e.AminoAcid)
 }
 
+// Stats denotes a set of stats computed from a genebank file used to update a table's weights
+type Stats struct {
```

Comment:
I think the `Stats` only make sense in the context of the genBank file we are using to update the weights, so I made it so we return it in the `UpdateWeightsWithSequence` function, and not tied to the codon table. We may update the table weights with multiple sequences, so not sure we can make it part of the `TranslationTable` struct?

Reply:
> We may update the table weights with multiple sequences

We will almost certainly be updating with multiple sequences